analytic_plan_sequence/models/analytic_plan.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountAnalyticPlan(models.Model):
+    _inherit = "account.analytic.plan"
+    _order = "sequence, complete_name asc"
+
+    sequence = fields.Integer(default=10)
+
+    @api.model
+    def get_relevant_plans(self, **kwargs):
+        res = super().get_relevant_plans(**kwargs)
+        if not res:
```

Comment:
I think this way is clearer, just a minor change.
```suggestion
        if res:
            plans = {p.id: p for p in self.browse([d["id"] for d in res])}
            for d in res:
                 d["sequence"] = plans[d["id"]].sequence
```

Reply:
I prefer not to use ladder-style code.